src/llm/wmt_data_utils.py

```diff
@@ -135,6 +137,11 @@ def _collate_fn(batch, source, target, tokenizers):
     tokenized_target_batch = [
         token.ids for token in tokenizers[target].encode_batch(target_batch)
     ]
+    # shift the target tokens by 1 to the right side by adding the <s> token at the beginning.
+    tokenized_target_batch = [
+        [tokenizers[target].token_to_id("<s>")] + token
+        for token in tokenized_target_batch
+    ]
```

Comment:
The comment is not clear to me.

All we are doing here is add the `<s>` special token to our target sequence.

Reply:
because we are adding <s> we have shifted the output tokens by 1 unit position. do you want me to change the comment?

Reply:
I think yes. It confuses me with the fact that shifting token by 1 is used for autoregressive models (decoder only)